src/nft/pages/asset/Asset.tsx

```diff
@@ -52,34 +49,20 @@ const Asset = () => {
         shouldLogImpression
       >
         {asset && collection ? (
```

Comment:
Asset and collection would be empty `[]` because of the useMemo which return truthy, a better solution would be to memo in the graphql hook and return the loading state earlier